react/features/base/ui/components/GlobalStyles.web.tsx

```diff
@@ -12,10 +12,12 @@ import { commonStyles } from '../constants';
 function GlobalStyles() {
     const { theme } = useStyles();
 
-    return (<MUIGlobalStyles
-        styles = {{
-            ...commonStyles(theme)
-        }} />);
+    return (
+        <MUIGlobalStyles
+            styles = {{
+                ...commonStyles(theme)
```

Comment:
can't we remove the extra object here? seems pointless right?

Reply:
I think we can